QUEST.tsv

```diff
@@ -1895,7 +1895,7 @@ QUEST_20150428_001898	The Letter that is Ripping into Pieces
 QUEST_20150428_001899	Ripped Diary
 QUEST_20150428_001900	The Last Record
 QUEST_20150428_001901	$The Abandoned Diary
-QUEST_20150428_001902	$The Goddess left many things to the first Paladin: {nl}the threat of the demon king, Gesti, and the mission to protect. {nl}She also mentioned that the savior would come.
+QUEST_20150428_001902	$The Goddess left many things to the first Paladin: {nl}The threat of the Demon Queen, Gesti, and the mission to protect. {nl}She also mentioned that the savior would come.
```

Comment:
How so? The title "Demon Queen" is more fitting for Gesti since:

1. She is a female;
2. The original is more closer to the meaning king or queen.